src/Raven.Server/Documents/Handlers/CountersHandler.cs

```diff
@@ -109,6 +107,7 @@ public override int Execute(DocumentsOperationContext context)
                                 LoadDocument();
                                  _database.DocumentsStorage.CountersStorage.PutCounterFromReplication(context, kvp.Key,
                                     operation.CounterName, operation.ChangeVector, operation.Delta);
+                                LastChangeVector = operation.ChangeVector;
```

Comment:
This is wrong and may reset the change vector back, actually

Reply:
We need to _only_ allow it if this is an update, not if this is already merged / conflict with the existing one

Reply:
done